nuxeo-core/nuxeo-core-event/src/test/java/org/nuxeo/ecm/core/work/WorkManagerTest.java

```diff
@@ -180,17 +199,16 @@ public void testWorkManagerConfig() throws Exception {
     public void testWorkManagerWork() throws Exception {
         MetricsTracker tracker = new MetricsTracker();
         tracker.assertDiff(0, 0, 0, 0);
-        int duration = 3000; // ms
-        SleepWork work = new SleepWork(duration, false);
+        SleepWork work = new SleepWork(getDurationMillis());
```

Comment:
Previously it was 3s now you set it to 200ms, why ?
I am afraid this will create randomness depending on CI load.

Reply:
Because these tests were taking quite long and I try to reduce execution time of tests I work on, 30s each time and the 3 hours we wait on every TP may one day get shorter.
Took it back to 3s.

Reply:
Still in this test, I can see no reason to wait that long.